### PR TITLE
Log relevant kubernetes events with each deployed API

### LINKED ISSUE #833: Export Kubernetes event logs
Requirements:

* output as json to stdout
* massage labels to match what is expected by Cortex
* Optional (or separate ticket): Add kubernetes event logs for the randomly chosen pod in `cortex logs` command

Possible tools:

* https://github.com/opsgenie/kubernetes-event-exporter (can be logged to stdout, not sure about structure)
* on GKE it ships with one by default (it has two containers: `gke.gcr.io/event-exporter` and `gke.gcr.io/prometheus-to-sd`, but that probably just can ship to stackdriver

### PR DESCRIPTION
**Pull request by miguelvr**
closes #833 

---

checklist:

- [x] run `make test` and `make lint`
- [x] test manually (i.e. build/push all images, restart operator, and re-deploy APIs)
- [x] update docs and add any new files to `summary.md` (view in [gitbook](https://cortex-labs.gitbook.io/staging/-MOmCGMADSRNQahK3Kox/) after merging)

### CODE REVIEW

File: build/images.sh [hunk 1]
```diff
@@ -50,7 +51,6 @@ non_dev_images_cluster=(
   "tensorflow-serving-cpu"
   "tensorflow-serving-gpu"
   "cluster-autoscaler"
-  "operator"
```
Comment (deliahu) [inline build/images.sh:53]: nit: can be reverted

File: charts/templates/event-exporter.yaml [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+              labels:
+                apiKind: RealtimeAPI
+        - match:
+            - receiver: "stdout"
+              labels:
+                apiKind: BatchAPI
+        - match:
+            - receiver: "stdout"
+              labels:
+                apiKind: TaskAPI
```
Comment (deliahu) [inline charts/templates/event-exporter.yaml:46]: consider: `isAPI: true`

File: images/event-exporter/Dockerfile [hunk 3]
```diff
@@ -0,0 +1 @@
+FROM opsgenie/kubernetes-event-exporter:0.9
```
Comment (deliahu) [inline images/event-exporter/Dockerfile:1]: Add to `versions.md`

**Comment by miguelvr** [comment]
@deliahu addressed all the issues

**Review by deliahu — APPROVED** [review]

**Pull request merged**